<h1 style='color: green; font-size: 36px; font-weight: bold;'>Data Science - Regressão Linear</h1>

# <font color='red' style='font-size: 30px;'>Simulador do modelo para preços de aluguel em SP</font>
<hr style='border: 2px solid red;'>

## Simulador interativo para Jupyter

In [1]:
# Importando bibliotecas
from ipywidgets import widgets, HBox, VBox
from IPython.display import display
import numpy as np
import pickle

# Carregando o modelo
modelo = open('modelo_preco_total_aluguel','rb')
lm_new = pickle.load(modelo)
modelo.close()

# Criando os controles do formulário
area = widgets.Text(description="Área (m²)", style={'description_width': 'initial'})
qtd_quartos = widgets.Text(description="Quantidade de quartos", style={'description_width': 'initial'})
qtd_garagens = widgets.Text(description="Quantidade de garagens", style={'description_width': 'initial'})
cod_bairro = widgets.Text(description="O código do bairro (Verificar tabela)", style={'description_width': 'initial'})
apartamento = widgets.Text(description="É apartamento? (0 ou 1)", style={'description_width': 'initial'})
casa = widgets.Text(description="É Casa? (0 ou 1)", style={'description_width': 'initial'})
casa_condominio = widgets.Text(description="É casa em condomínio? (0 ou 1)", style={'description_width': 'initial'})
studio_kitnet = widgets.Text(description="É Studio ou Kitnet? (0 ou 1)", style={'description_width': 'initial'})

botao = widgets.Button(description="Simular")

# Posicionando os controles
left = VBox([area, qtd_quartos, qtd_garagens, cod_bairro])
right = VBox([apartamento, casa, casa_condominio, studio_kitnet])
inputs = HBox([left, right])

# Função de simulação
def simulador(sender):
    entrada=[[
                np.log(float(area.value) + 1) if area.value else 0,
                np.log(float(qtd_quartos.value) + 1) if qtd_quartos.value else 0,
                np.log(float(qtd_garagens.value) + 1) if qtd_garagens.value else 0,
                int(cod_bairro.value if cod_bairro.value else 0),
                int(apartamento.value if apartamento.value else 0),
                int(casa.value if casa.value else 0),
                int(casa_condominio.value if casa_condominio.value else 0),
                int(studio_kitnet.value if studio_kitnet.value else 0)
             ]]
    
    print('Preço total do aluguel = R$ %.2f' % np.exp(lm_new.predict(entrada)[0]))
    
# Atribuindo a função "simulador" ao evento click do botão
botao.on_click(simulador)    

Preço total do aluguel = R$ 2840.53


c:\Users\NaímeAiubST-One\anaconda3\Lib\site-packages\sklearn\base.py:439: UserWarning: X does not have valid feature names, but LinearRegression was fitted with feature names
  warnings.warn(


In [2]:
display(inputs, botao)

Button(description='Simular', style=ButtonStyle())

#### Tabela para consulta do bairro pelo seu respectivo código **todos os nomes estão em ordem alfabetica, em minúsculo, sem acentos e concatenados**. Ex: Alto da Boa Vista = altodaboavista
##### para facilitar, é possível usar o 'ctrl+F' para procurar mais rápido, respeitando as regras citadas acima.

|Códigos|Bairros|
|---|---|
|0|aclimacao|
|1|adalgisa |
|2|aguabranca|
|3|aguafria|
|4|aguafunda|
|5|aguarasa|
|6|alphaville|
|7|altodaboavista|
|8|altodalapa|
|9|altodamooca|
|10|altodopari|
|11|americanopolis|
|12|anchieta|
|13|aricanduva|
|14|arturalvim|
|15|ayrosa|
|16|bandeiras|
|17|barrafunda|
|18|barrobranco(zonanorte)|
|19|belaalianca|
|20|belavista|
|21|belem|
|22|belenzinho|
|23|bethavillei|
|24|boacava|
|25|boavista|
|26|bolsaodointerlagos|
|27|bomretiro|
|28|bonfim|
|29|bortolandia|
|30|bosquedasaude|
|31|bras|
|32|brasilandia|
|33|brooklin|
|34|brooklinnovo|
|35|brooklinpaulista|
|36|burgopaulista|
|37|bussocaba|
|38|butanta|
|39|cambuci|
|40|campanario|
|41|campestre|
|42|campininha|
|43|campobelo|
|44|campogrande|
|45|camposeliseos|
|46|cangaiba|
|47|canhema|
|48|caninde|
|49|capeladosocorro|
|50|carandiru|
|51|casabranca|
|52|casaverde|
|53|casaverdealta|
|54|casaverdemedia|
|55|catumbi|
|56|caxingui|
|57|centro|
|58|centrocapital|
|59|centrocomercialjubran|
|60|centroempresarialtambore|
|61|centrohistoricodesaopaulo|
|62|cerqueiracesar|
|63|chacaraagrindus|
|64|chacarabelenzinho|
|65|chacaracalifornia|
|66|chacaracruzeirodosul|
|67|chacaradovovo|
|68|chacarainglesa|
|69|chacaraitaim|
|70|chacaramafalda|
|71|chacarameyer|
|72|chacaramontealegre|
|73|chacaranossasenhoradobomconselho|
|74|chacarasantoantonio|
|75|chacarasantoantonio(zonaleste)|
|76|chacarasantoantonio(zonasul)|
|77|chacarasaojoao|
|78|chacaraseisdeoutubro|
|79|choramenino|
|80|cidadeademar|
|81|cidadeantonioestevaodecarvalho|
|82|cidadebrasil|
|83|cidadecontinental|
|84|cidadedasflores|
|85|cidadedomitila|
|86|cidadedosbandeirantes|
|87|cidadedutra|
|88|cidadeintercap|
|89|cidadejardim|
|90|cidadelider|
|91|cidadeluz|
|92|cidademaedoceu|
|93|cidademaia|
|94|cidademoncoes|
|95|cidadenovasaomiguel|
|96|cidadepatriarca|
|97|cidadesaofrancisco|
|98|cidadesaomateus|
|99|cidadesatelitesantabarbara|
|100|cidadevargas|
|101|cipava|
|102|cityamerica|
|103|citybussocaba|
|104|cocaia|
|105|colonia(zonaleste)|
|106|conceicao|
|107|conjuntocityjaragua|
|108|conjuntohabitacionalpadremanoeldanobrega|
|109|conjuntohabitacionalteotoniovilela|
|110|conjuntopromorarsapopemba|
|111|conjuntoresidencialbutanta|
|112|conjuntoresidencialjardimcanaa|
|113|conjuntoresidencialjosebonifacio|
|114|conjuntoresidencialvistaverde|
|115|consolacao|
|116|continental|
|117|cupece|
|118|cursino|
|119|empresarial18doforte|
|120|engenheirogoulart|
|121|ermelinomatarazzo|
|122|fazendaaricanduva|
|123|fazendamorumbi|
|124|ferreira|
|125|freguesiadoo|
|126|fundacao|
|127|gopouva|
|128|guaianazes|
|129|guaiauna|
|130|guapira|
|131|helenamaria|
|132|higienopolis|
|133|hortoflorestal|
|134|i.a.p.i|
|135|imirim|
|136|indianopolis|
|137|industrialautonomistas|
|138|institutodeprevidencia|
|139|interlagos|
|140|ipiranga|
|141|itaberaba|
|142|itaimbibi|
|143|itaquaciara|
|144|itaquera|
|145|jabaquara|
|146|jacana|
|147|jaguara|
|148|jaguare|
|149|jaguaribe|
|150|jaragua|
|151|jardim|
|152|jardimadhemardebarros|
|153|jardimadriana|
|154|jardimadutora|
|155|jardimaeroporto|
|156|jardimagu|
|157|jardimaida|
|158|jardimalfredo|
|159|jardimalmanara|
|160|jardimalmeidaprado|
|161|jardimalvorada(zonaoeste)|
|162|jardimalzira|
|163|jardimamaralina|
|164|jardimamerica|
|165|jardimampliacao|
|166|jardimanaliafranco|
|167|jardimanalucia|
|168|jardimanarosa|
|169|jardimandarai|
|170|jardimangela(zonaleste)|
|171|jardimanny|
|172|jardimantartica|
|173|jardimaricanduva|
|174|jardimarize|
|175|jardimarizona|
|176|jardimarpoador|
|177|jardimatalibaleonel|
|178|jardimaurelia|
|179|jardimavelino|
|180|jardimbandeirantes(zonanorte)|
|181|jardimbarbosa|
|182|jardimbaruch|
|183|jardimbarueri|
|184|jardimbelavista|
|185|jardimbelem|
|186|jardimbelgica|
|187|jardimboavista(zonaoeste)|
|188|jardimbomclima|
|189|jardimbonfiglioli|
|190|jardimbotucatu|
|191|jardimbrasil|
|192|jardimbrasil(zonanorte)|
|193|jardimbrasil(zonasul)|
|194|jardimbrasilia(zonaleste)|
|195|jardimbrasilia(zonanorte)|
|196|jardimcabore|
|197|jardimcachoeira|
|198|jardimcampanario|
|199|jardimcampogrande|
|200|jardimcaner|
|201|jardimcanhema|
|202|jardimcaravelas|
|203|jardimcarlu|
|204|jardimcasablanca|
|205|jardimcastelo|
|206|jardimcatanduva|
|207|jardimceleste|
|208|jardimcentenario|
|209|jardimcidadepirituba|
|210|jardimcidalia|
|211|jardimcirino|
|212|jardimcity|
|213|jardimclimax|
|214|jardimcocaia|
|215|jardimcolombo|
|216|jardimcolorado|
|217|jardimconsorcio|
|218|jardimcordeiro|
|219|jardimcotiana|
|220|jardimcristal|
|221|jardimcristinalice|
|222|jardimd'abril|
|223|jardimdacampina|
|224|jardimdagloria|
|225|jardimdamamae|
|226|jardimdanfer|
|227|jardimdasacacias|
|228|jardimdasaude|
|229|jardimdasbandeiras|
|230|jardimdasesmeraldas|
|231|jardimdasflores|
|232|jardimdasgracas|
|233|jardimdaslaranjeiras|
|234|jardimdasmaravilhas|
|235|jardimdasnacoes|
|236|jardimdasperdizes|
|237|jardimdasvertentes|
|238|jardimdelorenzo|
|239|jardimdenise|
|240|jardimdinorah|
|241|jardimdivinolandia|
|242|jardimdocarmo|
|243|jardimdocolegio(zonanorte)|
|244|jardimdomar|
|245|jardimdombosco|
|246|jardimdonasinha|
|247|jardimdopapai|
|248|jardimdoscamargos|
|249|jardimdosestados|
|250|jardimdoslagos|
|251|jardimdosprados|
|252|jardimdotiro|
|253|jardimdourado|
|254|jardimdracena|
|255|jardimelvira|
|256|jardimernestina|
|257|jardimesmeralda|
|258|jardimester|
|259|jardimesteryolanda|
|260|jardimeuropa|
|261|jardimfelicidade(zonaoeste)|
|262|jardimfilhosdaterra|
|263|jardimflordamontanha|
|264|jardimfontedomorumbi|
|265|jardimfranca|
|266|jardimfreigalvao|
|267|jardimgildamaria|
|268|jardimgloria|
|269|jardimgracinda|
|270|jardimgraziela|
|271|jardimguairaca|
|272|jardimguanca|
|273|jardimguarapiranga|
|274|jardimguarau|
|275|jardimguarulhos|
|276|jardimgumercindo|
|277|jardimhaiadocarrao|
|278|jardimhelena|
|279|jardimhelian|
|280|jardimhercilia|
|281|jardimibitirama|
|282|jardimimperador(zonaleste)|
|283|jardimindependencia|
|284|jardimindependencia(saopaulo)|
|285|jardiminga|
|286|jardimipanema(zonasul)|
|287|jardimiporanga|
|288|jardimiracema|
|289|jardimiris|
|290|jardimitapeva|
|291|jardimiva|
|292|jardimivana|
|293|jardimjabaquara|
|294|jardimjacana|
|295|jardimjamaica|
|296|jardimjapao|
|297|jardimjaqueline|
|298|jardimjau(zonaleste)|
|299|jardimjoaoxxiii|
|300|jardimjua|
|301|jardimjulieta|
|302|jardimjussara|
|303|jardimkuabara|
|304|jardimlallo|
|305|jardimlarsaopaulo|
|306|jardimlasvegas|
|307|jardimleonor|
|308|jardimleonormendesdebarros|
|309|jardimlibano|
|310|jardimlideranca|
|311|jardimlondrina|
|312|jardimlosangeles|
|313|jardimlourdes|
|314|jardimlourdes(zonasul)|
|315|jardimluanda|
|316|jardimlucia|
|317|jardimluisa|
|318|jardimmaia|
|319|jardimmangalot|
|320|jardimmaraba|
|321|jardimmarajoara|
|322|jardimmariaaparecida|
|323|jardimmariaduarte|
|324|jardimmariaestela|
|325|jardimmarialuiza|
|326|jardimmariarita|
|327|jardimmariarosa|
|328|jardimmarilia|
|329|jardimmariliza|
|330|jardimmarilu|
|331|jardimmaringa|
|332|jardimmaristela|
|333|jardimmimar|
|334|jardimmirante|
|335|jardimmiriam|
|336|jardimmodelo|
|337|jardimmonjolo|
|338|jardimmontealegre|
|339|jardimmonteazul|
|340|jardimmontekemel|
|341|jardimmontelibano|
|342|jardimmontreal|
|343|jardimmoreira|
|344|jardimmorroverde|
|345|jardimmorumbi|
|346|jardimmunhozjunior|
|347|jardimnadir|
|348|jardimnordeste|
|349|jardimnorma|
|350|jardimnossasenhoraaparecida|
|351|jardimnossolar|
|352|jardimnovacidade|
|353|jardimnovagermania|
|354|jardimnovataboao|
|355|jardimnovosantoamaro|
|356|jardimnovotaboao|
|357|jardimodete|
|358|jardimolinda|
|359|jardimolympia|
|360|jardimoriental|
|361|jardimpalmares(zonasul)|
|362|jardimpalmira|
|363|jardimpanorama(zonaleste)|
|364|jardimparaiso|
|365|jardimparamount|
|366|jardimparaventi|
|367|jardimparquemorumbi|
|368|jardimpatente|
|369|jardimpatentenovo|
|370|jardimpaulista|
|371|jardimpaulistano|
|372|jardimpazini|
|373|jardimpedrojosenunes|
|374|jardimpenha|
|375|jardimpereiraleite|
|376|jardimperi|
|377|jardimperiperi|
|378|jardimpetropolis|
|379|jardimpinhal|
|380|jardimpiqueroby|
|381|jardimpiracuama|
|382|jardimpiratininga|
|383|jardimpirituba|
|384|jardimplanalto|
|385|jardimpopular|
|386|jardimprevidencia|
|387|jardimprimavera(zonanorte)|
|388|jardimprimavera(zonasul)|
|389|jardimprimeirodemaio(chacarafazendinha)|
|390|jardimpromissao|
|391|jardimprudencia|
|392|jardimraposotavares|
|393|jardimregina|
|394|jardimreginaalice|
|395|jardimregis|
|396|jardimrizzo|
|397|jardimroberto|
|398|jardimrosadefranca|
|399|jardimrosamaria|
|400|jardimrosana|
|401|jardimsabara|
|402|jardimsantabarbara|
|403|jardimsantacecilia|
|404|jardimsantaclara|
|405|jardimsantacruz|
|406|jardimsantacruz(campogrande)|
|407|jardimsantacruz(sacoma)|
|408|jardimsantaedwiges(capeladosocorro)|
|409|jardimsantaedwirges|
|410|jardimsantaefigenia|
|411|jardimsantaemilia|
|412|jardimsantahelena|
|413|jardimsantaines|
|414|jardimsantamaria|
|415|jardimsantamena|
|416|jardimsantamonica|
|417|jardimsantarosa|
|418|jardimsantaterezinha|
|419|jardimsantaterezinha(zonaleste)|
|420|jardimsantoamaro|
|421|jardimsantoantoninho|
|422|jardimsantoantonio|
|423|jardimsantoelias|
|424|jardimsantosdumont|
|425|jardimsaobento|
|426|jardimsaocaetano|
|427|jardimsaofrancisco(zonaleste)|
|428|jardimsaogabriel|
|429|jardimsaojoao(jaragua)|
|430|jardimsaojose|
|431|jardimsaojose(saomateus)|
|432|jardimsaojose(zonanorte)|
|433|jardimsaojudastadeu|
|434|jardimsaoluis|
|435|jardimsaopaulo|
|436|jardimsaopaulo(zonanorte)|
|437|jardimsaosaverio|
|438|jardimsaosilvestre|
|439|jardimsarah|
|440|jardimsatelite|
|441|jardimseckler|
|442|jardimsilvestre|
|443|jardimsilvia|
|444|jardimsoniamaria|
|445|jardimsulsaopaulo|
|446|jardimtabatinga|
|447|jardimtaboao|
|448|jardimtango|
|449|jardimtaquaral|
|450|jardimteresa|
|451|jardimterezopolis|
|452|jardimtestae|
|453|jardimtiete|
|454|jardimtijuco|
|455|jardimtoscana|
|456|jardimtranquilidade|
|457|jardimtresmarias|
|458|jardimtrussardi|
|459|jardimtupanci|
|460|jardimubirajara|
|461|jardimumarizal|
|462|jardimumuarama|
|463|jardimutinga|
|464|jardimvaleria|
|465|jardimvazani|
|466|jardimveracruz|
|467|jardimvergueiro|
|468|jardimvergueiro(sacoma)|
|469|jardimvilacarrao|
|470|jardimvilaformosa|
|471|jardimvilagalvao|
|472|jardimvilamariana|
|473|jardimvistaalegre|
|474|jardimvistalinda|
|475|jardimvitoriaregia|
|476|jardimvitoriaregia(zonaoeste)|
|477|jardimvivan|
|478|jardimzaira|
|479|jardins|
|480|jd.riopequeno|
|481|jdtranquilidade|
|482|jordanopolis|
|483|jurubatuba|
|484|km18|
|485|lajeado|
|486|lapa|
|487|lapadebaixo|
|488|larsaopaulo|
|489|lauzanepaulista|
|490|liberdade|
|491|limao|
|492|loteamentocityjaragua|
|493|luz|
|494|macedo|
|495|mandaqui|
|496|maranhao|
|497|maua|
|498|melvilleempresarialieii|
|499|melvilleempresarialii|
|500|mirandopolis|
|501|moema|
|502|moinhovelho|
|503|mooca|
|504|morrodosingleses|
|505|morumbi|
|506|mutinga|
|507|nossasenhoradoo|
|508|novaaldeinha|
|509|novagerti|
|510|novagerty|
|511|novapiraju|
|512|novoosasco|
|513|olimpico|
|514|osvaldocruz|
|515|pacaembu|
|516|padroeira|
|517|panamby|
|518|paradainglesa|
|519|paradaxvdenovembro|
|520|paraiso|
|521|paraisodomorumbi|
|522|paraisopolis|
|523|pari|
|524|parquealvesdelima|
|525|parquearariba|
|526|parquearturalvim|
|527|parqueassuncao|
|528|parquebairromorumbi|
|529|parqueboturussu|
|530|parquebristol|
|531|parquecasadepedra|
|532|parquececap|
|533|parquecisper|
|534|parquecolonial|
|535|parquecontinental|
|536|parquecontinentali|
|537|parquecontinentalii|
|538|parquecruzeirodosul|
|539|parquecruzeirodosul(vilaformosa)|
|540|parquedalapa|
|541|parquedamooca|
|542|parquedasnacoes|
|543|parquedaspaineiras|
|544|parquedavilaprudente|
|545|parquedoestado|
|546|parquedomorumbi|
|547|parquedosbancarios|
|548|parquedosprincipes|
|549|parqueeduchaves|
|550|parqueesmeralda|
|551|parquefongaro|
|552|parqueguarani|
|553|parqueimperial|
|554|parqueindependencia|
|555|parqueindustrialtomasedson|
|556|parqueipe|
|557|parqueitaberaba|
|558|parquejabaquara|
|559|parquejoaoramalho|
|560|parquemandaqui|
|561|parquemandi|
|562|parquemariadomitila|
|563|parquemarialuiza|
|564|parquemontealegre|
|565|parquemonteirosoares|
|566|parquemunhoz|
|567|parquenacoesunidas|
|568|parquenovomundo|
|569|parqueoratorio|
|570|parquepaineiras|
|571|parquepanamericano|
|572|parquepenha|
|573|parqueperuche|
|574|parqueramosfreitas|
|575|parquereboucas|
|576|parqueregina|
|577|parquerenatomaia|
|578|parqueresidencialdalapa|
|579|parqueresidencialjulia|
|580|parquesantaluzia|
|581|parquesantamadalena|
|582|parquesantoantonio|
|583|parquesantosdumont|
|584|parquesaodomingos|
|585|parquesaofrancisco|
|586|parquesaojorge|
|587|parquesaolucas|
|588|parquesaoluis|
|589|parquesevilha|
|590|parquetomassaraiva|
|591|parquevitoria|
|592|pauliceia|
|593|pedreira|
|594|penhadefranca|
|595|perdizes|
|596|pestana|
|597|picanco|
|598|pinheiros|
|599|piqueri|
|600|piratininga|
|601|pirituba|
|602|planaltopaulista|
|603|pompeia|
|604|pontegrande|
|605|pontepequena|
|606|portaldosgramados|
|607|presidentealtino|
|608|quartaparada|
|609|quintadapaineira|
|610|quitauna|
|611|raposotavares|
|612|realparque|
|613|recantodasrosas|
|614|recantomorumbi|
|615|recantoparaiso|
|616|republica|
|617|residencialtres(tambore)|
|618|retiromorumbi|
|619|riopequeno|
|620|rolinopolis|
|621|rudgeramos|
|622|sacoma|
|623|santacecilia|
|624|santaefigenia|
|625|santaifigenia|
|626|santaines|
|627|santamaria|
|628|santana|
|629|santapaula|
|630|santarita|
|631|santateresinha|
|632|santaterezinha|
|633|santoamaro|
|634|santoantonio|
|635|saodomingos|
|636|saojoaoclimaco|
|637|saojose|
|638|saojudas|
|639|saolucas|
|640|saomiguelpaulista|
|641|saopedro|
|642|sapopemba|
|643|saude|
|644|se|
|645|siciliano|
|646|silveira|
|647|sitiodafigueira|
|648|sitiodomandaqui|
|649|sitiodomorro|
|650|sitiodopiqueri|
|651|sitiomorrogrande|
|652|sitiopinheirinho|
|653|socorro|
|654|suico|
|655|suisso|
|656|sumare|
|657|sumarezinho|
|658|superquadramorumbi|
|659|taboao|
|660|tambore|
|661|tatuape|
|662|terceiradivisaodeinterlagos|
|663|tremembe|
|664|tucuruvi|
|665|umuarama|
|666|usinapiratininga|
|667|utinga|
|668|varzeadabarrafunda|
|669|varzeadoglicerio|
|670|veleiros|
|671|veloso|
|672|vilaadalgisa|
|673|vilaaguafunda|
|674|vilaalbano|
|675|vilaalbertina|
|676|vilaalexandria|
|677|vilaalgusta|
|678|vilaalianca|
|679|vilaalice|
|680|vilaalpina|
|681|vilaaltodesantoandre|
|682|vilaalzira|
|683|vilaamalia(zonaleste)|
|684|vilaamalia(zonanorte)|
|685|vilaamelia|
|686|vilaamerica|
|687|vilaanadir|
|688|vilaanastacio|
|689|vilaandrade|
|690|vilaanglobrasileira|
|691|vilaanhanguera|
|692|vilaantonieta|
|693|vilaantonina|
|694|vilaantonio|
|695|vilaantoniodossantos|
|696|vilaaraguaia|
|697|vilaarapua|
|698|vilaarcadia|
|699|vilaaricanduva|
|700|vilaarriete|
|701|vilaarruda|
|702|vilaassuncao|
|703|vilaaugusta|
|704|vilaaurora(zonanorte)|
|705|vilaazevedo|
|706|vilababilonia|
|707|vilabancaria|
|708|vilabandeirantes|
|709|vilabarbosa|
|710|vilabarreto|
|711|vilabarros|
|712|vilabaruel|
|713|vilabasileia|
|714|vilabastos|
|715|vilabeatriz|
|716|vilabela|
|717|vilabelavista(zonanorte)|
|718|vilabertioga|
|719|vilaboacava|
|720|vilaboavista|
|721|vilabonilha|
|722|vilabonilhanova|
|723|vilabrasil|
|724|vilabrasilandia|
|725|vilabrasilina|
|726|vilabrasiliomachado|
|727|vilabremen|
|728|vilabruna|
|729|vilabuarque|
|730|vilabuenosaires|
|731|vilabutanta|
|732|vilacachoeira|
|733|vilacalifornia|
|734|vilacalifornia(zonasul)|
|735|vilacamilopolis|
|736|vilacaminhodomar|
|737|vilacampanela|
|738|vilacampesina|
|739|vilacampestre|
|740|vilacampogrande|
|741|vilacanero|
|742|vilacapitaorabelo|
|743|vilacaraguata|
|744|vilacarbone|
|745|vilacardosofranco|
|746|vilacarlosdecampos|
|747|vilacarmem|
|748|vilacarmosina|
|749|vilacarolina|
|750|vilacarrao|
|751|vilacastelo|
|752|vilacatupia|
|753|vilacavaton|
|754|vilaceleste|
|755|vilacentenario|
|756|vilaclarice|
|757|vilaclaudia|
|758|vilaclementino|
|759|vilacleonice|
|760|vilacomercial|
|761|vilaconceicao|
|762|vilacongonhas|
|763|vilaconstanca|
|764|vilaconstancia|
|765|vilacontinental|
|766|vilacordeiro|
|767|vilacosmopolita|
|768|vilacostamelo|
|769|vilacristalia|
|770|vilacruzdasalmas|
|771|vilacruzeiro|
|772|vilacuruca|
|773|viladalila|
|774|viladalva|
|775|viladapaz|
|776|viladasaude|
|777|viladasbandeiras|
|778|viladasbelezas|
|779|viladasmerces|
|780|viladaspalmeiras|
|781|viladeodoro|
|782|viladinorah|
|783|viladionisia|
|784|viladiva|
|785|viladiva(zonaleste)|
|786|viladiva(zonanorte)|
|787|viladivinapastora|
|788|viladobosque|
|789|viladocastelo|
|790|viladoencontro|
|791|viladomitila|
|792|viladompedroi|
|793|viladompedroii|
|794|viladora|
|795|viladosandrades|
|796|vilaede|
|797|vilaeldizia|
|798|vilaelvira|
|799|vilaelze|
|800|vilaema|
|801|vilaemir|
|802|vilaendres|
|803|vilaengenhonovo|
|804|vilaerna|
|805|vilaernesto|
|806|vilaesperanca|
|807|vilaesplanada|
|808|vilaester|
|809|vilaester(zonanorte)|
|810|vilaeuthalia|
|811|vilafachini|
|812|vilafatima|
|813|vilafernandes|
|814|vilafiatlux|
|815|vilafidalgo|
|816|vilafidelisribeiro|
|817|vilafirmianopinto|
|818|vilafiuza|
|819|vilafloresta|
|820|vilaflorida|
|821|vilaformosa|
|822|vilafranci|
|823|vilafranciscomatarazzo|
|824|vilafriburgo|
|825|vilagabriel|
|826|vilagalvao|
|827|vilagea|
|828|vilagerminal|
|829|vilagertrudes|
|830|vilagilda|
|831|vilagomes|
|832|vilagomescardim|
|833|vilagopouva|
|834|vilagraciosa|
|835|vilagranada|
|836|vilaguaca|
|837|vilaguarani|
|838|vilaguarani(zonaleste)|
|839|vilaguarani(zonasul)|
|840|vilaguarani(zsul)|
|841|vilaguedes|
|842|vilaguilherme|
|843|vilaguilhermina|
|844|vilaguiomar|
|845|vilagumercindo|
|846|vilagustavo|
|847|vilahamburguesa|
|848|vilaharmonia|
|849|vilahebe|
|850|vilahelena|
|851|vilaherminia|
|852|vilahulda|
|853|vilahumaita|
|854|vilainah|
|855|vilaindependencia|
|856|vilaindiana|
|857|vilaindustrial|
|858|vilainglesa|
|859|vilainvernada|
|860|vilaiorio|
|861|vilaipojuca|
|862|vilairmaosarnoni|
|863|vilaisa|
|864|vilaisolinamazzei|
|865|vilaivone|
|866|vilajacui|
|867|vilajaguara|
|868|vilalageado|
|869|vilalais|
|870|vilaleonor|
|871|vilaleopoldina|
|872|vilalibanesa|
|873|vilaliviero|
|874|vilalondrina|
|875|vilalucia|
|876|vilaluciaelvira|
|877|vilalucinda|
|878|vilamadalena|
|879|vilamafra|
|880|vilamangalot|
|881|vilamaracana|
|882|vilamarari|
|883|vilamargarida|
|884|vilamaria|
|885|vilamariaalta|
|886|vilamariabaixa|
|887|vilamariana|
|888|vilamariatrindade|
|889|vilamariazelia|
|890|vilamaricy|
|891|vilamarieta|
|892|vilamarina|
|893|vilamarizamazzei|
|894|vilamarte|
|895|vilamascote|
|896|vilamatiilde|
|897|vilamatilde|
|898|vilamazzei|
|899|vilamedeiros|
|900|vilamendes|
|901|vilamercedes|
|902|vilamesquita|
|903|vilametalurgica|
|904|vilamilton|
|905|vilamira|
|906|vilamiriam|
|907|vilamoinhovelho|
|908|vilamontealegre|
|909|vilamonumento|
|910|vilamoraes|
|911|vilamoreira|
|912|vilamorellato|
|913|vilamorse|
|914|vilamorumbi|
|915|vilanair|
|916|vilanancy|
|917|vilanatalia|
|918|vilanelson|
|919|vilanhocune|
|920|vilanilo|
|921|vilanivi|
|922|vilanogueira|
|923|vilanossasenhoradefatima|
|924|vilanova|
|925|vilanovaalba|
|926|vilanovacachoeirinha|
|927|vilanovacaledonia|
|928|vilanovacarolina|
|929|vilanovaconceicao|
|930|vilanovacuruca|
|931|vilanovadasbelezas|
|932|vilanovaesperanca|
|933|vilanovamanchester|
|934|vilanovamazzei|
|935|vilanovapauliceia|
|936|vilanovasantaluzia|
|937|vilanovasavoia|
|938|vilanovayork|
|939|vilansradefatima|
|940|vilaolimpia|
|941|vilaolinda|
|942|vilaoratorio|
|943|vilaosasco|
|944|vilapaiva|
|945|vilapalmares|
|946|vilapalmeiras|
|947|vilaparanagua|
|948|vilaparapua|
|949|vilaparquejabaquara|
|950|vilapauliceia|
|951|vilapaulista|
|952|vilapaulistania|
|953|vilapaulosilas|
|954|vilapedrabranca|
|955|vilapedromoreira|
|956|vilapenteado|
|957|vilapereirabarreto|
|958|vilapereiracerca|
|959|vilapicinin|
|960|vilapierina|
|961|vilapirajussara|
|962|vilapires|
|963|vilapirituba|
|964|vilapita|
|965|vilaplana|
|966|vilaplanalto|
|967|vilapolopoli|
|968|vilapompeia|
|969|vilaportugal|
|970|vilaportuguesa|
|971|vilapousoalegre|
|972|vilaprado|
|973|vilapraia|
|974|vilaprel|
|975|vilaprimavera|
|976|vilaprincesaisabel|
|977|vilaprincipedegales|
|978|vilaprogredior|
|979|vilaprogresso|
|980|vilaprogresso(zonanorte)|
|981|vilaprudente|
|982|vilaramos|
|983|vilare|
|984|vilaregentefeijo|
|985|vilaregina|
|986|vilarenata|
|987|vilario|
|988|vilariobranco|
|989|vilariodejaneiro|
|990|vilaromana|
|991|vilaromano|
|992|vilaromero|
|993|vilaroque|
|994|vilarosalia|
|995|vilaruibarbosa|
|996|vilasabrina|
|997|vilasacaduracabral|
|998|vilasalete|
|999|vilasantacatarina|
|1000|vilasantaclara|
|1001|vilasantaedwiges|
|1002|vilasantaisabel|
|1003|vilasantaluzia|
|1004|vilasantamaria|
|1005|vilasantana|
|1006|vilasantateresa|
|1007|vilasantateresa(zonaleste)|
|1008|vilasantateresa(zonasul)|
|1009|vilasantaterezinha|
|1010|vilasantaterezinha(zonanorte)|
|1011|vilasantavirginia|
|1012|vilasantista|
|1013|vilasantoantonio|
|1014|vilasantoestefano|
|1015|vilasantoestevao|
|1016|vilasantos|
|1017|vilasaodomingos|
|1018|vilasaofrancisco|
|1019|vilasaofrancisco(zonaleste)|
|1020|vilasaofrancisco(zonasul)|
|1021|vilasaogeraldo|
|1022|vilasaojoao|
|1023|vilasaojoaobatista|
|1024|vilasaojose|
|1025|vilasaojose(ipiranga)|
|1026|vilasaojudastadeu|
|1027|vilasaoluis|
|1028|vilasaoluis(zonaoeste)|
|1029|vilasaoluiz(valparaizo)|
|1030|vilasaonicolau|
|1031|vilasaopaulo|
|1032|vilasaopedro|
|1033|vilasaoricardo|
|1034|vilasaosilvestre|
|1035|vilasaovicente|
|1036|vilasargentojosedepaula|
|1037|vilascarpelli|
|1038|vilaserralheiro|
|1039|vilasilveira|
|1040|vilasilvia|
|1041|vilasiqueira(zonanorte)|
|1042|vilasirene|
|1043|vilasocorro|
|1044|vilasofia|
|1045|vilasonia|
|1046|vilasouza|
|1047|vilasuzana|
|1048|vilatalarico|
|1049|vilatijuco|
|1050|vilatiradentes|
|1051|vilatolstoi|
|1052|vilatrabalhista|
|1053|vilatramontano|
|1054|vilauberabinha|
|1055|vilauniao|
|1056|vilauniao(zonaleste)|
|1057|vilavalparaiso|
|1058|vilavenditti|
|1059|vilavera|
|1060|vilavermelha|
|1061|vilavirginia|
|1062|vilavivaldi|
|1063|vilayara|
|1064|vilayolanda|
|1065|vilazamataro|
|1066|vilazat|
|1067|vilazelina|
|1068|vilazilda|
